### ISSUE TITLE
Square brackets converted to round brackets

### ISSUE THREAD

**Issue by drydenboss** [HessiJames/soundkonverter#25]
SoundKonverter converts the \\[ \\] into parentheses after the format has been selected. I was hoping to be able to use square brackets but it fails in this way. This happens at every place I try to use them. Now I'll have to manually change this using a custom script. Let me see if I can do that at all ;-). But in the meantime, I was hoping for this to work better.

This is version 2.1.1. Using Kubuntu 14.10.

**Comment by HessiJames** [comment]
Are you trying to write to a FAT or NTFS file system or do you have the option "Advanced" -> "Always use FAT compatible output file names" enabled?

**Comment by drydenboss** [comment]
You are correct! I have that option enabled. But the weird thing is that I was always able to select these characters in Windows? For filenames. I don't think \[\] are reserved characters in Windows? I have many files (or had) with these characters in them.
